build/craft_release.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# This script is used to create a new release rock image from the included template
+set -eu
+
+RO_SCRIPT_DIR="$( dirname "${BASH_SOURCE[0]}")"
+RO_REPO_DIR=$(dirname ${RO_SCRIPT_DIR})
+RO_VERSIONS=${RO_SCRIPT_DIR}/../versions.txt
+
+function usage() {
+    if [[ -z ${COREDNS_GIT_DIR+x} ]]; then
+        echo "COREDNS_GIT_DIR is not set" > /dev/stderr
+        echo "   Clone with 'git clone --bare --filter=blob:none --no-checkout https://github.com/coredns/coredns.git /tmp/coredns.git'" > /dev/stderr
+        echo "   Re-run with 'COREDNS_GIT_DIR=/tmp/coredns.git $0'" > /dev/stderr
+        exit 1
+    fi
+}
+
+
+function check_dependencies(){
+    for cmd in yq jq git envsubst; do
+        if ! command -v $cmd &> /dev/null; then
+            echo "$cmd could not be found" > /dev/stderr
+            exit 1
+        fi
+    done
+}
+
+
+function create_rockcrafts(){
+    # Get the current releases from existing rockcraft yamls
+    if [[ -v GITHUB_WORKSPACE ]]; then
+        echo "::group::Create rockcrafts"
+    fi
+
+    rm -rf ${RO_VERSIONS}
+    for rockcraft in $(find ${RO_REPO_DIR} -name 'rockcraft.yaml'); do
+        echo $(yq '.version' $rockcraft) >> ${RO_VERSIONS}
```

Comment:
echo not needed

Reply:
so?  it seems this is required.  Without it 
```sh
yq '.version' $rockcraft >> ${RO_VERSIONS}
```


i get an error:
```
Error: write /dev/stdout: bad file descriptor
```

Reply:
i think it's due to snap confinement when yq is installed via snap